web/frontend/src/components/base.vue

```diff
@@ -79,17 +78,20 @@ const setDirection = (dir: Directions) => {
   direction.value = dir;
 };
 
-const stop = () => {
+const stop = async () => {
+  const bc = new BaseClient(props.client, props.name, { requestLogger: rcLogConditionally });
```

Comment:
Nit: Can we use a more descriptive variable name like `baseClient`.

CC: @micheal-parks surprised the linter didn't catch this one?

Reply:
I think we only catch single letter var usage. I'd be open to bump that up to two letters. But yeah, `baseClient` would be the better name here.

Reply:
Sounds good, Ill update the client name for board and motor as well since we use the two letter convention there.